torch/nn/modules/module.py

```diff
@@ -94,6 +94,16 @@ def __setstate__(self, state: Dict):
 _global_is_full_backward_hook: Optional[bool] = None
 _global_forward_pre_hooks: Dict[int, Callable] = OrderedDict()
 _global_forward_hooks: Dict[int, Callable] = OrderedDict()
+_has_global_hooks: bool = False
+
+def _update_has_global_hooks():
```

Comment:
Shouldn't this also be called when handle.remove() is called?
If you don't care about accurate tracking, you can just set it to True and that's it?

Reply:
it is called by handle.remove()

tagging the code below where it is called.